### DISCUSSION TITLE
playright bug

### DISCUSSION THREAD

**Discussion by zarigata** [stitionai/devika#504] [Q&A]
i cant seen to pass the playright screen, it si stuck at the playright part, ive tried everything and it is frozen at this part, i cant install it

║ Looks like Playwright was just installed or updated.       ║
║ Please run the following command to download new browsers: ║
║                                                            ║
║     playwright install                                     ║
║                                                            ║
║ <3 Playwright Team

**Comment by Polygons1** [comment]
have you tried running `playwright install --with-deps`?
